core/src/style/style.cpp

```diff
@@ -138,11 +140,16 @@ void Style::onBeginDrawFrame(const View& _view, Scene& _scene) {
 
     // Set Map Position
     if (m_dirtyViewport) {
-        const auto& mapPos = _view.getPosition();
         m_shaderProgram->setUniformf("u_resolution", _view.getWidth(), _view.getHeight());
-        m_shaderProgram->setUniformf("u_map_position", mapPos.x, mapPos.y, _view.getZoom());
     }
 
+    const auto& mapPos = _view.getPosition();
+    m_shaderProgram->setUniformf("u_map_position", mapPos.x, mapPos.y, _view.getZoom());
+    m_shaderProgram->setUniformMatrix3f("u_normalMatrix", glm::value_ptr(_view.getNormalMatrix()));
+    m_shaderProgram->setUniformf("u_meters_per_pixel", _view.pixelsPerMeter());
```

Comment:
is it meters per pixel or pixels per meter? if the names are correct then shouldn't this should be inverse?
`1.0f/_view.pixelsPerMeter()`.